test/development/acceptance/ReactRefreshRegression.test.ts

```diff
@@ -286,10 +286,21 @@ describe('ReactRefreshRegression', () => {
     expect(await session.hasRedbox()).toBe(true)
 
     const source = await session.getRedboxSource()
-    expect(source.split(/\r?\n/g).slice(2).join('\n')).toMatchInlineSnapshot(`
-      "> 1 | export default function () { throw new Error('boom'); }
-          |                                   ^"
-    `)
+
+    if (process.env.TURBOPACK) {
+      expect(source).toMatchInlineSnapshot(`
+        "pages/index.js (0:54) @ __TURBOPACK__default__export__
+
+          1 | export default function () { throw new Error('boom'); }"
```

Comment:
The source location still looks wrong to me. 

`0:54` doesn't make sense because the error is on line `1` right?

Reply:
🤔 @wbinnssmith wasn't there similar case you had to deal with offsets?